core/templates/dev/head/services/alertsServiceSpec.js

```diff
@@ -17,184 +17,184 @@
-      expect(alertsService.messages[0].content).toBe('Info 2');
-      expect(alertsService.messages[1].content).toBe('Success 1');
+      expect(AlertsService.messages[0].content).toBe('Info 2');
+      expect(AlertsService.messages[1].content).toBe('Success 1');
     });
 
     it('should delete a message (with duplicates)', function() {
       var message = 'Info 1';
       // Info Message to be deleted
-      alertsService.addInfoMessage(message);
+      AlertsService.addInfoMessage(message);
       // Add a few other messages
-      alertsService.addInfoMessage('Info 2');
-      alertsService.addSuccessMessage('Success 1');
-      alertsService.addInfoMessage(message);
+      AlertsService.addInfoMessage('Info 2');
+      AlertsService.addSuccessMessage('Success 1');
+      AlertsService.addInfoMessage(message);
 
-      expect(alertsService.messages.length).toBe(4);
-      alertsService.deleteMessage({
+      expect(AlertsService.messages.length).toBe(4);
+      AlertsService.deleteMessage({
         type: 'info',
         content: message
       });
-      expect(alertsService.messages.length).toBe(2);
+      expect(AlertsService.messages.length).toBe(2);
 
       // Search for the message
       var found = false;
-      for (var i = 0; i < alertsService.messages.length; i++) {
-        if (alertsService.messages[i].content === message &&
-            alertsService.messages[i].type === 'info') {
+      for (var i = 0; i < AlertsService.messages.length; i++) {
+        if (AlertsService.messages[i].content === message &&
+            AlertsService.messages[i].type === 'info') {
           found = true;
         }
       }
       expect(found).toBe(false);
-      expect(alertsService.messages[0].content).toBe('Info 2');
-      expect(alertsService.messages[1].content).toBe('Success 1');
+      expect(AlertsService.messages[0].content).toBe('Info 2');
+      expect(AlertsService.messages[1].content).toBe('Success 1');
     });
 
     it('should not add more than 10 messages', function() {
       var message = 'Info ';
       for (var i = 1; i < 15; i++) {
-        alertsService.addInfoMessage(message + i);
+        AlertsService.addInfoMessage(message + i);
       }
-      alertsService.addSuccessMessage('Success 1');
-      expect(alertsService.messages.length).toBe(10);
+      AlertsService.addSuccessMessage('Success 1');
+      expect(AlertsService.messages.length).toBe(10);
     });
 
     it('should clear all the messages', function() {
-      alertsService.addInfoMessage('Info 1');
-      alertsService.addInfoMessage('Info 2');
-      alertsService.addSuccessMessage('Success 1');
-      alertsService.clearMessages();
-      expect(alertsService.messages.length).toBe(0);
+      AlertsService.addInfoMessage('Info 1');
+      AlertsService.addInfoMessage('Info 2');
+      AlertsService.addSuccessMessage('Success 1');
+      AlertsService.clearMessages();
+      expect(AlertsService.messages.length).toBe(0);
     });
   });
 });
```

Comment:
Let's also rename the spec files. Could you rename `alertsServiceSpec.js` to `AlertsServiceSpec.js`. Sorry, I think I forgot to mention that in the instructions for the issue :(